test/project/partials/formats/erb/.gitkeep

```diff
@@ -0,0 +1 @@
+# Resolver breaks when files and dir's have the same name
```

Comment:
Basically all without extensions break. Agreeing in theory but what I want to test is basically:
```
assert_find "formats/erb.html.erb", "formats/erb"
```